### PR TITLE
feat(console): add new step to activate SMTP provider

### LINKED ISSUE #7929: SMTP templates. Show activate provider as another optional step
### Preflight Checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions
- [X] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Describe your problem

@eliobischof suggested in [Discord](https://discord.com/channels/927474939156643850/1229787441485844544/1229842480527315004) that it'd good if you can activate the SMTP provider in a similar way as the new IDP wizard works. 

### Describe your ideal solution

Here's the screenshot for a new Google IDP

image

I'd add the 'active this SMTP provider' as an optional step once you create the provider (the dialog would not be closed and this step would be shown). 

I agree and I missed that option when developing SMTP templates... so I open an issue to track this feature. 

Of course, feel free to assign this issue to me.

### Version

_No response_

### Environment

Self-hosted

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by doncicuto**
# Which Problems Are Solved

- In #7929 it was detected that it would be better to show the activate/deactivate action for a SMTP configuration in the wizard as for some users it'd not be intuitive that the SMTP provider must be activated so Zitadel can use it to send notifications.

# How the Problems Are Solved

- When a new SMTP provider is added or updated, the wizard has a new step that allow us to activate or deactivate the provider configured in the previous step. The following video shows the new wizard:

https://github.com/zitadel/zitadel/assets/30386061/178234d6-73dc-4719-af0b-1d6f19bf3f7d

# Additional Context

- Closes #7929

### CODE REVIEW

File: console/src/assets/i18n/bg.json [hunk 1]
```diff
@@ -2246,7 +2246,16 @@
         "CURRENT_DESC_TITLE": "Това са вашите SMTP настройки",
         "PROVIDER_SETTINGS": "Настройки на SMTP доставчик",
         "SENDER_SETTINGS": "Настройки на изпращача",
-        "TEST_SETTINGS": "Тествайте настройките на SMTP"
+        "TEST_SETTINGS": "Тествайте настройките на SMTP",
+        "NEXT_STEPS": "Следващи стъпки",
+        "ACTIVATE": {
+          "TITLE": "Активирайте вашия SMTP доставчик",
+          "DESCRIPTION": "Zitadel не може да използва този SMTP доставчик за изпращане на известия, докато не го активирате. Ако активирате този доставчик, всеки друг доставчик, който е бил активен, сега ще бъде деактивиран."
```
Comment (eliobischof) [inline console/src/assets/i18n/bg.json:2253]:
```suggestion
          "DESCRIPTION": "ZITADEL не може да използва този SMTP доставчик за изпращане на известия, докато не го активирате. Ако активирате този доставчик, всеки друг доставчик, който е бил активен, сега ще бъде деактивиран."
```

others too

File: console/src/app/modules/smtp-provider/smtp-provider.component.html [hunk 2]
```diff
@@ -143,6 +143,66 @@ <h2>
...
+      <form>
+        <ng-template matStepLabel>{{ 'SMTP.CREATE.STEPS.NEXT_STEPS' | translate }}</ng-template>
+        <cnsl-info-section *ngIf="!isActive">
+          <div class="title-row">
+            <div class="left">
+              <h2 class="title">{{ 'SMTP.CREATE.STEPS.ACTIVATE.TITLE' | translate }}</h2>
+              <div>
+                <a mat-icon-button card-actions mat-icon-button rel="noreferrer" target="_blank">
+                  <mat-icon class="next-icon">info_outline</mat-icon>
+                </a>
```
Comment (eliobischof) [inline console/src/app/modules/smtp-provider/smtp-provider.component.html:156]: This doesn't point anywhere, right?
Let's point it to https://zitadel.com/docs/guides/manage/console/default-settings#smtp

Same in the deactivate section

**Review by eliobischof — CHANGES_REQUESTED** [review]
I love it @doncicuto :rocket: 

Just some minor change requests...

**Comment by doncicuto** [comment]
Sure @eliobischof, thanks for your review I'll prepare the changes ASAP

File: console/src/assets/i18n/bg.json (hunk 1 above)
Comment (doncicuto) [reply console/src/assets/i18n/bg.json]: Done :smile:

File: console/src/app/modules/smtp-provider/smtp-provider.component.html (hunk 2 above)
Comment (doncicuto) [reply console/src/app/modules/smtp-provider/smtp-provider.component.html]: Done :smile:

**Review by eliobischof — APPROVED** [review]

**Review by eliobischof — CHANGES_REQUESTED** [review]
@doncicuto can you have a look at the e2e tests, please?

**Comment by doncicuto** [comment]
Sorry @eliobischof I totally missed the e2e tests, my bad :flushed:

**Review by eliobischof — APPROVED** [review]

**Pull request merged**